src-docs/spec/setUpdateListener.md

```diff
@@ -29,6 +29,41 @@ Each `update` which is passed to the callback comes with the following propertie
 
 - `update.summary`: optional, short text, shown beside icon (see [`sendUpdate()`])
 
+Example:
+
+```js
+let myDocumentState = "";
+const initialPendingUpdatesHandledPromise = window.webxdc.setUpdateListener(
+  (update) => {
+    // Remember that the listener is invoked for
+    // your own `window.webxdc.sendUpdate()` calls as well!
+
+    // Dummy document update logic.
+    // Yours might be more complex,
+    // such as applying a chess move to the board.
+    myDocumentState = myDocumentUpdate;
+
+    const areMoreUpdatesPending = update.serial !== update.max_serial;
+    if (!areMoreUpdatesPending) {
+      renderDocument();
+    }
+  }
+);
+
+initialPendingUpdatesHandledPromise.then(() => {
+  renderDocument();
+});
```

Comment:
this is not used?

Reply:
This is useful when nobody has sent any updates yet.